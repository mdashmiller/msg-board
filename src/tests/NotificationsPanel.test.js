// import React from 'react'
// import ReactDOM from 'react-dom'
// import renderer from 'react-test-renderer'
// // import Enzyme, { shallow } from 'enzyme'
// // import Adapter from 'enzyme-adapter-react-16'

// import NotificationsPanel from '../components/dashboard/NotificationsPanel'

// // Enzyme.configure({ adapter: new Adapter() })

// describe('<NotificationsPanel /> rendering', () => {

// 	it('renders without crashing', () => {
// 	    const div = document.createElement('div')
// 	    ReactDOM.render(<NotificationsPanel />, div)
// 	    ReactDOM.unmountComponentAtNode(div)
// 	})

// 	test('has a valid snapshot', () => {
// 		const component = renderer.create(<NotificationsPanel />)
// 		let tree = component.toJSON()
// 		expect(tree).toMatchSnapshot()
// 	})
// })
