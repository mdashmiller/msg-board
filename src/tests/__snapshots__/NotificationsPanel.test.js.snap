// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotificationsPanel /> rendering has a valid snapshot 1`] = `
<div
  className="hide-on-large-only notifications-panel"
>
  <div
    className="section notifications-content"
  >
    <h5
      className="note-panel-title"
    >
      Notifications
    </h5>
  </div>
</div>
`;
